### PR TITLE
Add python API codeowners

### PR DESCRIPTION
**Pull request by akuporos**
FYI: @nkogteva

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -14,6 +14,7 @@ azure-pipelines.yml  @openvinotoolkit/openvino-admins
 
 # IE Core:
 /inference-engine/  @openvinotoolkit/openvino-ie-maintainers
+/inference-engine/ie_bridges/python @akuporos @nkogteva
```
Comment (azhogov) [inline CODEOWNERS:17]: probably @openvinotoolkit/openvino-tools-maintainers can be used here

File: CODEOWNERS [hunk 2]
```diff
@@ -63,4 +64,4 @@ azure-pipelines.yml  @openvinotoolkit/openvino-admins
 /ngraph/  @openvinotoolkit/openvino-ngraph-maintainers
 
 # Tools
-/tools/  @openvinotoolkit/openvino-tools-maintainers
+/tools/  @openvinotoolkit/openvino-tools-maintainers @akuporos
```
Comment (azhogov) [inline CODEOWNERS:67]: you is a member of @openvinotoolkit/openvino-tools-maintainers now

File: CODEOWNERS (hunk 1 above)
Comment (akuporos) [reply CODEOWNERS:17]: What about created team @openvinotoolkit/openvino-ie-python-api-maintainers?
It'll be deleted?

File: CODEOWNERS (hunk 2 above)
Comment (akuporos) [reply CODEOWNERS]: fixed

File: CODEOWNERS (hunk 1 above)
Comment (akuporos) [reply CODEOWNERS]: I used @openvinotoolkit/openvino-ie-python-api-maintainers

**Pull request merged**